### ISSUE TITLE
Log4j2 log NULL

### ISSUE THREAD

**Issue by rosolko** [reportportal/logger-java-log4j#1]
# Steps
1. Create logger instance:
`private static final Logger LOGGER = LogManager.getLogger("Default");`
2. Log message, where alert message is **'Sign in to continue'** :
`logger.info("Check that an alert appears : '{0}'", alert.getText());`

# Expected:
Log same message to reportportal and console: 
`15:21:05 [main] INFO  - Check that an alert appears : 'Sign in to continue'`

# Actual:
Log to console correct. 
But log to reportportal looks like (with level color highlight): 
`NULL`

## Screenshot:
![image](https://cloud.githubusercontent.com/assets/11458319/21766640/b9a20b22-d680-11e6-8b9f-b7778316495f.png)

**Comment by avarabyeu** [comment]
@rosolko Alex, seems like there is some incorrect usage of Log4j2
Here is documentation: https://logging.apache.org/log4j/2.0/manual/messages.html
If you use pattern in logging string, you don't need to specify parameter index. So in your, it should look like:
```
logger.info("Check that an alert appears : '{0}'", alert.getText());
```
Feel free to reopen issue if i'm wrong

**Comment by avarabyeu** [comment]
Fixed in 2.6.1
